internal/tools/aws/constants.go

```diff
@@ -203,6 +203,15 @@ const (
 	// schemas allowed in a Posgres multitenant RDS database cluster.
 	DefaultRDSMultitenantDatabasePostgresCountLimit = 300
 
+	// DefaultRDSMultitenantPGBouncerDatabasePostgresCountLimit is the maximum
+	// number of schemas allowed in a Posgres multitenant RDS database cluster
+	// with a PGBouncer proxy.
+	DefaultRDSMultitenantPGBouncerDatabasePostgresCountLimit = 1000
```

Comment:
@gabrieljackson Is this basically the max number of installations per db cluster?
 Write

Reply:
Yeah, it's completely arbitrary and just a value to get us started.